runtime/manta-pc/src/lib.rs

```diff
@@ -802,16 +806,33 @@ impl_runtime_apis! {
 		}
 	}
 
-	#[cfg(feature = "try-runtime")]
-	impl frame_try_runtime::TryRuntime<Block> for Runtime {
```

Comment:
This hook for the try-runtime upgrade of the frame pallets.
Is it not compiling or something ?

Reply:
This hook is back.